homeassistant/components/vesync/switch.py

```diff
@@ -53,28 +48,43 @@ class VeSyncSwitchHA(SwitchDevice):
     def __init__(self, plug):
         """Initialize the VeSync switch device."""
         self.smartplug = plug
-        self._current_power_w = None
-        self._today_energy_kwh = None
 
     @property
     def unique_id(self):
         """Return the ID of this switch."""
-        return self.smartplug.cid
+        if isinstance(self.smartplug.sub_device_no, int):
+            return (self.smartplug.cid + str(self.smartplug.sub_device_no))
+        else:
+            return self.smartplug.cid
 
     @property
     def name(self):
         """Return the name of the switch."""
         return self.smartplug.device_name
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the device."""
+        attr = {}
+        attr['active_time'] = self.smartplug.active_time
```

Comment:
Active time is the number of minutes the device has been online.  This is not a time stamp

Reply:
Will this time update just because time passes? We don't allow relative times in the state machine, since it will spam the state machine with not needed state changes, just due to time passing.

Reply:
Bump.

Reply:
I missed that comment, sorry.  It's the amount of time a switch has been turned on.  I don't really see a use for it, it's gone!